todo_app/flask_config.py

```diff
@@ -1,8 +1,19 @@
 import os
 
-
 class Config:
     """Base configuration variables."""
     SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
SECRET_KEY isn't actually needed anymore as it was for encrypting session cookies